assets/swagger.json

```diff
@@ -3171,7 +3171,7 @@
         "parameters": [
           {
             "type": "string",
-            "description": "URL is the URL that this credentials matches to",
+            "description": "URL is the URL that these credentials matches to",
```

Comment:
```suggestion
            "description": "URL is the URL that these credentials match",
```

or possibly "URL is the URL to which these credentials match"

Reply:
I prefer the latter formulation, I'll take that one. Thanks!